core/templates/dev/head/pages/collection-editor-page/editor-tab/collection-node-creator.directive.ts

```diff
@@ -55,11 +55,6 @@ angular.module('oppia').directive('collectionNodeCreator', [
             SearchExplorationsBackendApiService, SiteAnalyticsService,
             INVALID_NAME_CHARS) {
           var ctrl = this;
-          ctrl.collection = CollectionEditorStateService.getCollection();
-          ctrl.newExplorationId = '';
-          ctrl.newExplorationTitle = '';
-          ctrl.searchQueryHasError = false;
-
           var CREATE_NEW_EXPLORATION_URL_TEMPLATE = '/create/<exploration_id>';
```

Comment:
Looks like we don't use this. Can we remove it?

Reply:
Done.